package.json

```diff
@@ -1,69 +1,66 @@
-		"deploy:cart": "cd ./apps/cart && npm run build && CLOUDFLARE_ACCOUNT_ID=XXX npx wrangler pages publish dist --project-name=qwik-dream-cart",
-		"deploy:hero": "cd ./apps/hero && npm run build && CLOUDFLARE_ACCOUNT_ID=XXX npx wrangler pages publish dist --project-name=qwik-dream-hero",
-		"deploy:host": "cd ./apps/host && npm run build && CLOUDFLARE_ACCOUNT_ID=XXX npx wrangler pages publish dist --project-name=qwik-dream-demo",
-		"deploy:menu": "cd ./apps/menu && npm run build && CLOUDFLARE_ACCOUNT_ID=XXX npx wrangler pages publish dist --project-name=qwik-dream-menu",
-		"deploy:product": "cd ./apps/product && npm run build && CLOUDFLARE_ACCOUNT_ID=XXX npx wrangler pages publish dist --project-name=qwik-dream-product",
-		"deploy:reviews": "cd ./apps/reviews && npm run build && CLOUDFLARE_ACCOUNT_ID=XXX npx wrangler pages publish dist --project-name=qwik-dream-reviews",
-		"preview:cart": "npm run --prefix ./apps/cart preview",
-		"preview:hero": "npm run --prefix ./apps/hero preview",
-		"preview:host": "npm run --prefix ./apps/host preview",
-		"preview:menu": "npm run --prefix ./apps/menu preview",
-		"preview:product": "npm run --prefix ./apps/product preview",
-		"preview:reviews": "npm run --prefix ./apps/reviews preview",
-		"dev:cart": "cd ./apps/cart && npm run dev",
-		"dev:hero": "cd ./apps/hero && npm run dev",
-		"dev:host": "cd ./apps/host && npm run dev",
-		"dev:menu": "cd ./apps/menu && npm run dev",
-		"dev:product": "cd ./apps/product && npm run dev",
-		"dev:reviews": "cd ./apps/reviews && npm run dev",
-		"dev": "run-p dev:* --print-label",
-		"preview": "run-p preview:* --print-label",
-		"deploy": "run-s deploy:* --print-label"
-	},
-	"repository": {
-		"type": "git",
-		"url": "git+https://github.com/gioboa/qwik-dream-demo.git"
-	},
-	"license": "ISC",
-	"bugs": {
-		"url": "https://github.com/gioboa/qwik-dream-demo/issues"
-	},
-	"homepage": "https://github.com/gioboa/qwik-dream-demo#readme",
-	"devDependencies": {
-		"@builder.io/qwik": "^0.15.2",
-		"@builder.io/qwik-city": "^0.0.128",
-		"@builder.io/sdk-qwik": "^0.0.36",
-		"@nrwl/cli": "15.3.0",
-		"@nrwl/js": "15.3.0",
-		"@nrwl/nx-cloud": "latest",
-		"@nrwl/workspace": "15.3.0",
-		"@tailwindcss/forms": "^0.5.3",
-		"@types/eslint": "8.4.10",
-		"@types/node": "latest",
-		"@typescript-eslint/eslint-plugin": "5.45.1",
-		"@typescript-eslint/parser": "5.45.1",
-		"autoprefixer": "^10.4.13",
-		"eslint": "8.29.0",
-		"eslint-plugin-qwik": "0.15.2",
-		"node-fetch": "3.3.0",
-		"npm-run-all": "^4.1.5",
-		"nx": "15.3.0",
-		"postcss": "^8.4.19",
-		"prettier": "2.8.1",
-		"tailwindcss": "^3.2.4",
-		"typescript": "4.9.4",
-		"vite": "3.2.5",
-		"vite-tsconfig-paths": "4.0.1",
-		"wrangler": "2.6.1"
-	}
+  "name": "qwik-dream-demo",
+  "version": "1.0.0",
+  "engines": {
+    "node": ">=18"
+  },
+  "description": "Qwik dream demo",
+  "private": true,
+  "scripts": {},
```

Comment:
Can you create few standard scripts please?

Reply:
@gioboa added scripts to serve, preview and deploy all apps